src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -456,8 +457,10 @@ function get_sysinfo_item_html($itemkey) {
 	//sort categories
 	ksort($data);
 	foreach ($data as $cat => $cat_data) {
-		// display title for this category
-		echo "<tr><th><strong>{$cat_data['cattitle']}</strong></th>\n<td>&nbsp;</td></tr>";
+		if (strlen($cat_data['cattitle']) > 0) {
+			// display title for this category
+			echo "<tr><th><strong>{$cat_data['cattitle']}</strong></th>\n<td>&nbsp;</td></tr>";
+		}
```

Comment:
Only display the cattitle if it has something in it. Handles the "no items to display" case when there is no cattitle. Needs to go back into master PR.